x/mint/types/constants.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+const (
+	SecondsPerMinute = 60
+	MinutesPerHour   = 60
+	HoursPerDay      = 24
+	DaysPerYear      = 365
+	SecondsPerYear   = SecondsPerMinute * MinutesPerHour * HoursPerDay * DaysPerYear // 31,536,000
```

Comment:
I'm not sure if you want to go down the rabbit hole of time but using a Gregorian calendar there is actually 365.2425 days in a year (because of the leap year) which means theres actually 31556952 seconds in a year. YMMV

Reply:
Related: https://github.com/cosmos/cosmos-sdk/issues/16009

I choose to use `365` because I value readability and don't think a high degree of precision is necessary for this calculation but I can update for higher precision with more documentation.

Reply:
fd0274ee7cda88cb9835619ffe0e272f903c87da